libs/sdk-flutter/lib/breez_sdk.dart

```diff
@@ -22,7 +23,7 @@ class BreezSDK {
 
   Stream<Payment> get paymentResultStream => _paymentResultStream.stream;
 
-  void initialize() {
+  void initialize({bool usesNativeNodeLogs = false}) {
```

Comment:
This doesn't sound clear to me. Also it requires documentation to help the user understand whey the logstream is not emitting any log events...
If we want to use the native API to listen to log stream then we should do it always and wire it to the flutter code using plugin.